commands.go

```diff
@@ -50,6 +51,11 @@ func initCommands(config *Config, services *disco.Disco) {
 		services.ForceHostServices(host, hostConfig.Services)
 	}
 
+	configDir, err := ConfigDir()
```

Comment:
Should this be using `cliconfig.ConfigDir()` instead? I'm not sure if we're intending to remove the top-level `config.go`eventually.

Reply:
Good point! I wasn't planning to do anything pro-active around the old compatibility wrappers in the `main` package for now, but since this is new code we might as well write it the new way from the start, rather than perpetuating the use of the old functions.